### PR TITLE
chore: support for testnet5

### PR DESCRIPTION
**Pull request by jinoosss**
## Descriptions

Add the testnet5 meta information.

### CODE REVIEW

**Review by onlyhyde — DISMISSED** [review]

**Comment by dongwon8247** [comment]
@jinoosss Before merging it, please add monikers for validators from [r/gnoland/valopers/v2/valopers.gno](https://test4.gno.land/r/gnoland/valopers/v2/valopers.gno)

File: public/resource/chains.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   {
-    "name": "Testnet 4",
-    "chainId": "test4",
-    "apiUrl": "https://test4.api.onbloc.xyz/v1",
-    "rpcUrl": "https://rpc.test4.gno.land:443",
-    "indexerUrl": "https://test4.indexer.onbloc.xyz"
+    "name": "Testnet 5",
+    "chainId": "test5",
+    "apiUrl": "https://test5.api.onbloc.xyz/v1",
+    "rpcUrl": "https://rpc.test5.gno.land:443",
```
Comment (dongwon8247) [inline public/resource/chains.json:6]: Let's use `https://test5.onbloc.xyz/` for now.

File: src/common/values/constant-value.ts [hunk 2]
```diff
@@ -4,3 +4,13 @@ export const RPC_URI = process.env.NEXT_PUBLIC_RPC_URI ?? '';
 export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_TRACKING_ID ?? '';
 
 export const DAY_TIME = 86_400_000 as const; // Day time: 24 * 60 * 60 * 1000
+
+export enum ChainType {
+  TESTNET5 = 'test5',
+  TESTNET4 = 'test4',
```
Comment (dongwon8247) [inline src/common/values/constant-value.ts:10]: We can depreciate all test4 info

**Review by dongwon8247 — CHANGES_REQUESTED** [review]

File: public/resource/chains.json (hunk 1 above)
Comment (jinoosss) [reply public/resource/chains.json]: I updated this PR.(https://github.com/onbloc/gnoscan/pull/121/commits/fd704f65ef58dc7c040c14e2b02f1127a2ece338)

File: src/common/values/constant-value.ts (hunk 2 above)
Comment (jinoosss) [reply src/common/values/constant-value.ts]: I removed it this commit (https://github.com/onbloc/gnoscan/pull/121/commits/a4a2dbdefaeaf1992946512a603abb75d374ad08)

**Review by dongwon8247 — APPROVED** [review]

**Review by dongwon8247 — APPROVED** [review]

**Pull request merged**